packages/styles/scss/components/ui-shell/header/_header.scss

```diff
@@ -137,6 +140,9 @@
     text-decoration: none;
     transition: border-color $duration-fast-02;
     user-select: none;
+    @include breakpoint-down('md') {
+      padding: 0 mini-units(2);
```

Comment:
Could we use the spacing tokens here? 

https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale